spec/requests/rack_attack_spec.rb

```diff
@@ -394,7 +394,7 @@
     context 'when the number of requests is under the limit' do
       it 'does not throttle the request' do
         (phone_setups_per_ip_limit - 1).times do
-          patch '/phone_setup', headers: { REMOTE_ADDR: '1.2.3.4' }
+          post '/phone_setup', headers: { REMOTE_ADDR: '1.2.3.4' }
```

Comment:
As mentioned in the extended commit message, this is fortunately not cause for concern, since throttling applies to any non-GET request:

https://github.com/18F/identity-idp/blob/e10b2638cc4c751c7d60fefa9de747683f9ee449/config/initializers/rack_attack.rb#L202